src/PeerReview/Types.hs

```diff
@@ -53,11 +57,11 @@ data ReviewStatus = Waiting
                   | Accepted
                   deriving (Show)
 
-data ReviewTask = ReviewTask
-    { rtContent :: Text
-    , rtTaskId  :: TaskID
-    , rtUserId  :: UserID
-    } deriving (Show)
+data Task = Task
+    { tId      :: TaskID
```

Comment:
Good point.
Let's split Task into Task and Submission in a PR that will also split the TaskSource and define interfaces.